### PR TITLE
OSDOCS-4231:Set default HAProxy maxconn value to 50000

### PR DESCRIPTION
**Pull request by skrthomas**
4.12+
Issue:
https://issues.redhat.com/browse/NE-1072

Link to docs preview:
https://51058--docspreview.netlify.app/openshift-enterprise/latest/networking/ingress-operator.html#nw-ingress-controller-configuration-parameters_configuring-ingress

Can you please give a QE review @ShudiLi :
- [ ] QE has approved this change.

### CODE REVIEW

**Review by jldohmann — COMMENTED** [review]
Mostly LGTM, I think [Ingress Controller is supposed to be capitalized](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#ingress-controller).

File: modules/nw-ingress-controller-configuration-parameters.adoc [hunk 1]
```diff
@@ -225,9 +225,9 @@ supports up to `64` threads. If this field is empty, the Ingress Controller uses
...
+* `maxConnections` specifies the maximum number of simultaneous connections that can be established per HAProxy process. Increasing this value allows each ingress controller pod to handle more connections at the cost of additional system resources. Permitted values are `0`, `-1`, any value within the range `2000` and `2000000`, or the field can be left empty. 
 
-** If this field is left empty or has the value `0`, the ingress controller will use the default value of `20000`. This value is subject to change in future releases.
+** If this field is left empty or has the value `0`, the ingress controller will use the default value of `50000`. This value is subject to change in future releases.
 
 ** If the field has the value of `-1`, then HAProxy will dynamically compute a maximum value based on the available `ulimits` in the running container. This process results in a large computed value that will incur significant memory usage compared to the current default value of `20000`.
```
Comment (frobware) [inline modules/nw-ingress-controller-configuration-parameters.adoc:232]: This line also mentions 20000 and should also be adjusted for 50000.

Comment (skrthomas) [reply modules/nw-ingress-controller-configuration-parameters.adoc:232]: Oof, sorry for missing this.

**Pull request merged**